action.yml

```diff
@@ -26,11 +27,34 @@ runs:
     env:
       INPUT_APP: ${{ inputs.app_name }}
       INPUT_TUTOR_VERSION: ${{ inputs.tutor_version }}
+  
+  - name: Prepare Tutor Nightly and Launch
+    if: ${{ inputs.tutor_version == 'nightly' }}
+    run: |
+      pip install virtualenv
+      virtualenv venv
+      source venv/bin/activate
+      git clone --branch=nightly --depth=1 https://github.com/overhangio/tutor.git
+      chmod 777 tutor -R
+      pip install -e "./tutor[full]"
+      TUTOR_ROOT="$(pwd)" tutor --version
+      TUTOR_ROOT="$(pwd)" tutor config save
+      TUTOR_ROOT="$(pwd)" tutor mounts add lms,cms,lms-worker,cms-worker:$(pwd)/$INPUT_APP:/openedx/$INPUT_APP
+      chmod 777 . -R
+      TUTOR_ROOT="$(pwd)" tutor images build openedx 
+      TUTOR_ROOT="$(pwd)" tutor local launch -I
```

Comment:
Since these commands repeat, I think we can avoid repetition by adding a step for it, similar to how it is here: https://github.com/eduNEXT/eox-tenant/blob/ae80464e2a38b36adb8e110c9511c66502b5f543/.github/workflows/integration_tutor_nightly.yml#L27-L35, what do you think?

Reply:
I thought about that, and the reason why I didn't split it was to minimize the repetition of the conditional to know if is running nightly and should activate the virtual env.

 However, I leave this one to discussion, I'm happy to hear other's opinion.

_Note_: As it is is almost the same, could be the same if we can guaranty that the image building is not necessary.

Reply:
Yes, it would be good to check if it is not necessary make build. So, we can add a step that is common for all tutor versions.